libpod/runtime.go

```diff
@@ -339,18 +340,36 @@ func getDefaultTmpDir() (string, error) {
 
 // SetXdgRuntimeDir ensures the XDG_RUNTIME_DIR env variable is set
 // containers/image uses XDG_RUNTIME_DIR to locate the auth file.
-func SetXdgRuntimeDir(val string) error {
+func SetXdgRuntimeDir() error {
 	if !rootless.IsRootless() {
 		return nil
 	}
-	if val == "" {
+
+	runtimeDir := os.Getenv("XDG_RUNTIME_DIR")
+
+	runtimeDirLinger, err := rootless.EnableLinger()
```

Comment:
Should mention this in the comment

Reply:
would you like a comment about what runtimeDirLinger is?

Reply:
I'd like something in the function's header comment to indicate that `SetXdgRuntimeDir()` also calls `EnableLinger()` - so we know where the call is happening from the docs.

Reply:
fixed and pushed